book/website/community-handbook/translation/translation-workflow.md

```diff
@@ -47,7 +47,7 @@ Make sure you read these guidelines before you start translating for the first t
 ### Create and update a glossary
 
 We strongly suggest setting a glossary through the localisation platform.
-You can read more about project glossaries in {ref}`Addin terms to the glossary<translation-hello-crowdin#adding-terms-to-the-glossary>`.
+You can read more about project glossaries in {ref}`Addin terms to the glossary<translation-hello-crowdin>`.
```

Comment:
Adding instead  of addin?

Reply:
ah! this removal is making the build stop. Please revert to the tag (or add the correct tag?)